cmd/boulder-wfe2/main.go

```diff
@@ -62,6 +69,75 @@ type config struct {
 	}
 }
 
+// loadCertificateChains processes the provided chainConfig of AIA Issuer URLs
+// and cert filenames. For each AIA issuer URL all of its cert filenames are
+// read, validated as PEM certificates, and concatinated together separated by
+// newlines. The combined PEM certificate chain contents for each are returned
+// in the results map, keyed by the AIA Issuer URL.
+func loadCertificateChains(chainConfig map[string][]string) (map[string]string, error) {
+	results := make(map[string]string, len(chainConfig))
+
+	// For each AIA Issuer URL we need to read the chain files
+	for aiaIssuerURL, chainfiles := range chainConfig {
+		var buffer bytes.Buffer
+
+		// There must be at least one chainfile specified
+		if len(chainfiles) == 0 {
+			return nil, berrors.MalformedError(
```

Comment:
There's no need to use `berrors.MalformedError` here because this isn't in the RPC call chain. Just use a regular `fmt.Errorf`. Same with instances below.

Reply:
Fixed in f3bd898